documentation/docs/client/threads_and_replies.md

```diff
@@ -0,0 +1,81 @@
+---
+id: clientThreadAndReplies
+title: Threads and Replies
+sidebar_position: 19
+---
+
+Threads and replies provide your users with a way to go into more detail about a specific topic. This can be very helpful to keep the conversation organized and reduce noise.
+
+## Creating a Thread
+
+To create a thread you simply send a message with a `parent_id`. Have a look at the example below:
+
+```kotlin
+val message = Message( 
+    text = "Hello there!", 
+    parentId = parentMessage.id, 
+) 
+ 
+// Send the message to the channel 
+channelClient.sendMessage(message).enqueue { result -> 
+    if (result.isSuccess) { 
+        val sentMessage = result.data() 
+    } else { 
+        // Handle result.error() 
+    } 
+}
+```
+
+> If you specify `show_in_channel`, the message will be visible both in a thread of replies as well as the main channel.
+
+Messages inside a thread can also have reactions, attachments and mention as any other message.
+
+## Retrieving Thread Messages
+
+When you read a channel you do not receive messages inside threads. The parent message includes the count of replies which it is usually what apps show as the link to the thread screen. Reading a thread and paginating its messages works in a very similar way as paginating a channel.
```

Comment:
`When you read a channel you do not receive messages inside threads.` - 🤔 are you sure about that? You should receive a message but it shouldn't be visible until you enter to the thread

Reply:
This text is from the CMS. Yep, I'm sure.